packages/ui-library/src/components/transaction-confirmation/index.tsx

```diff
@@ -31,23 +31,10 @@ import {
 } from '../../emojis';
 
 const StyledOverlay = styled.div`
-  ${({
-    theme: {
-      palette: { mode },
-      spacing,
-    },
-  }) => `
-    position: absolute;
-    top: 0;
-    bottom: 0;
-    left: 0;
-    right: 0;
-    z-index: 99;
-    padding: ${spacing(6)};
+  ${({ theme: { spacing } }) => `
     display: flex;
     flex-direction: column;
     gap: ${spacing(6)};
-    background-color: ${colors[mode].background.quarteryNoAlpha};
```

Comment:
sure we should remove this one?

Reply:
Yep. Components that used to be 'behind' are now not rendered